### ISSUE TITLE
[a11y]: Accessibility violation in Toggle and ComboBox Component

### ISSUE THREAD

**Issue by vaibhu444** [carbon-design-system/carbon#17163]
### Package

@carbon/react

### Package version

1.64.0-rc.0

### React version

18.2.0

### Automated testing tool and ruleset

IBM Accessibility Check npm package

### Description

here is the first violation in ComboBox:-
<img width="1728" alt="Screenshot 2024-08-14 at 10 10 28 AM" src="https://github.com/user-attachments/assets/4639ee62-a18b-49be-a518-1baca54136c7">
here is the Second violation in Toggle
<img width="1728" alt="Screenshot 2024-08-13 at 8 23 36 PM" src="https://github.com/user-attachments/assets/c6c92771-3b55-4ae5-a838-6bba21934cef">

### Reproduction/example

https://stackblitz.com/edit/github-rwldes?file=src%2FApp.jsx

### Steps to reproduce

here is live link:- link:- https://stackblitz.com/edit/github-rwldes?file=src%2FApp.jsx

### Suggested Severity

Severity 3 = User can complete task, and/or has a workaround within the user experience of a given component.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by heloiselui** [comment]
Thanks for opening this issue! I retested everything in Storybook and in a fresh StackBlitz, and I couldn’t reproduce the errors anymore. It looks like they were fixed in carbon/react v1.99. I'll close this issue, but If the problem still shows up for you, we can reopen this or open a new issue.
